stdlib/2and3/filecmp.pyi

```diff
@@ -0,0 +1,49 @@
+# Stubs for filecmp (Python 2/3)
+import sys
+from typing import AnyStr, Callable, Dict, Generic, Iterable, List, Optional, Sequence, Tuple
+
+_SupportsBool = object
+
+DEFAULT_IGNORES = ...  # type: List[str]
+
+def cmp(f1: AnyStr, f2: AnyStr, shallow: _SupportsBool = ...) -> bool: ...
```

Comment:
I believe I tested this manually and found that mixing str and bytes was bad in python3. I'll test again.